dnsutil/zone.go

```diff
@@ -0,0 +1,212 @@
+// Generate zonefiles.
+// This generates a zonefile that prioritizes beauty over efficiency.
+
+package dnsutil
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"sort"
+	"strings"
+
+	"github.com/miekg/dns"
+)
+
+type zoneGenData struct {
```

Comment:
Can you put this change in a separate PR? Still not sure I want it or not and in what form. We can then merge the other functions